src/ui/components/results/resultsheadercomponent.js

```diff
@@ -10,9 +13,10 @@ const DEFAULT_CONFIG = {
   resultsCountSeparator: '|',
   verticalURL: undefined,
   showChangeFilters: false,
-  removable: false, // TODO implement
+  removable: false,
   delimiter: '|',
-  isUniversal: false
+  isUniversal: false,
+  verticalKey: undefined
```

Comment:
Do we need this default? Aren't we always guaranteed to have a verticalKey?

Reply:
can get rid of it wasn't sure if we wanted to list out all possible config options or not